incognia_test.go

```diff
@@ -386,16 +398,17 @@ func (suite *IncogniaTestSuite) TestErrorsRegisterFeedback() {
 }
 
 func (suite *IncogniaTestSuite) TestSuccessRegisterPayment() {
-	transactionServer := suite.mockPostTransactionsEndpoint(token, postPaymentRequestBodyFixture, transactionAssessmentFixture)
+	transactionServer := suite.mockPostTransactionsEndpoint(token, postPaymentRequestBodyFixture, transactionAssessmentFixture, queryStringWithFalseEval)
```

Comment:
Shouldn't this query string be empty (or with true eval)? If we're sending false eval here, the expected response would be empty, but the `transactionAssessmentFixture` contains a non-empty assessment.